_data/iamspathan/iamspathan.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sohail Pathan # No longer than 18 characters
+institution: Lovely Professional University # no longer than 58 characters
+profile_pic: sohail.png # Relative path to the image you uploaded
+quote: "I'm going to graduate, I want to thank teachers, Google, Wikipedia, coffee, and copy paste for helping in this journey." # No longer than 100 characters
```

Comment:
Character count exceeds the limit of 100, you might want to modify this part and could you also please delete the double quotes (`"`) from both sides to guarantee the format remains the same.

Reply:
changes made in last commit